ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMap_Overrides/README.md

```diff
@@ -10,39 +10,42 @@ When taking a web map offline, you may adjust the data (such as layers or tiles)
 
 ## How to use the sample
 
-1. When the sample starts, you may be prompted to sign into arcgis.com.
-2. Click on "Generate Offline Map (Overrides)".
-3. Use the range slider to adjust the min/max levelIds to be taken offline for the Streets basemap.
-4. Use the spin-box to set the buffer radius for the streets basemap.
-5. Click the buttons to skip the System Valves and Service Connections layers.
-4. Use the combo-box to select the maximum flow rate for the features from the Hydrant layer.
-5. Use the check-box to skip the geometry filter for the water pipes features.
-6. Click "Start Job"
-7. Wait for the progress bar to indicate that the task has completed.
-8. You should see that the basemap does not display when you zoom out past a certain range and is padded around the original area of interest. The network dataset should extend beyond the target area. The System Valves and Service Connections should be omitted from the offline map and the Hydrants layer should contain a subset of the original features.
-
-# How it works
+1. Click on "Generate Offline Map (Overrides)".
+2. Use the range slider to adjust the min/max levelIds to be taken offline for the Streets basemap.
+3. Use the spin-box to set the buffer radius for the streets basemap.
+4. Click the buttons to skip the System Valves and Service Connections layers.
+5. Use the combo-box to select the maximum flow rate for the features from the Hydrant layer.
+6. Use the check-box to skip the geometry filter for the water pipes features.
+7. Click "Start Job"
+8. Wait for the progress bar to indicate that the task has completed.
```

Comment:
I liked your earlier revision in "Generate offline map", because yeah, it's not a bar--it's a view.
```suggestion
8. Wait for the progress view to indicate that the task has completed.
```